src/pks/transport/transport_ats_pk.cc

```diff
@@ -124,7 +124,6 @@ void Transport_ATS::Setup(const Teuchos::Ptr<State>& S)
   prev_saturation_key_ = Keys::readKey(*plist_, domain_, "previous saturation liquid", "prev_saturation_liquid");
   flux_key_ = Keys::readKey(*plist_, domain_, "water flux", "water_flux");
   permeability_key_ = Keys::readKey(*plist_, domain_, "permeability", "permeability");
-  tcc_key_ = Keys::readKey(*plist_, domain_, "concentration", "total_component_concentration");
```

Comment:
Why was this removed?  This key still needs a value, no?

Reply:
Oh, this should probably be:

tcc_key_ = key_ ?  Or maybe that already exists somewhere?

Reply:
Yes, tcc_key_ is initialized in constructor